sources/logdog/static/common

```diff
@@ -0,0 +1,15 @@
+sh containerd-config containerd config dump
```

Comment:
nit, because it's really a question about the existing command: do we need to do something to talk to the host-containerd specifically, rather than the containerd used for k8s?

Reply:
I'm not sure, both of these commands have the exact same output:

```
containerd --address /run/host-containerd/containerd.sock config dump
containerd --address /run/containerd/containerd.sock config dump
```

Both of these report:

```
version = 2
root = "/var/lib/containerd"
state = "/run/containerd"
plugin_dir = ""
disabled_plugins = ["io.containerd.internal.v1.opt", "io.containerd.snapshotter.v1.aufs", "io.containerd.snapshotter.v1.devmapper", "io.containerd.snapshotter.v1.native", "io.containerd.snapshotter.v1.zfs"]
required_plugins = []
oom_score = 0
imports = ["/etc/containerd/config.toml"]

[grpc]
  address = "/run/dockershim.sock"
  tcp_address = ""
  tcp_tls_cert = ""
  tcp_tls_key = ""
  uid = 0
  gid = 0
  max_recv_message_size = 16777216
  max_send_message_size = 16777216
etc...
```

It seems like both of these report the non-host-containerd config (not sure though). I'm hoping we can avoid blocking on this.

Reply:
You need `containerd --config /etc/host-containerd/config.toml config dump` to get the host-containerd config data.

Reply:
[aa7d592](https://github.com/bottlerocket-os/bottlerocket/compare/60e5d3c8e65c58170e8f72d944a80be11ea098e0..aa7d59202dad5161a9dc8aff3995f1fbe0d4c706)